projects/material-addons/src/lib/quick-list/quick-list.component.html

```diff
@@ -9,6 +9,7 @@ <h3>
     </mad-icon-button>
   </div>
 </ng-container>
-<mad-link-button *ngIf="!(readonly || readonly === '')" (click)="addItem()" [disabled]="!addPossible || !isAddAllowed()">
+<mad-link-button *ngIf="!(readonly || readonly === '')" [clickEvent]="addEventFunction"
```

Comment:
(clickEvent) does not solve the "global" issue that (click)-events on disabled mad-buttons still occur. Devs will always use (click) on buttons